packages/contractkit/src/test-utils/AlphabeticSequencer.ts

```diff
@@ -0,0 +1,14 @@
+const Sequencer = require('@jest/test-sequencer').default
+
+class AlphabeticSequencer extends Sequencer {
```

Comment:
Why do we need a sequencer? I would put some documentation about why & how it works... or what it solves

Reply:
By default jest uses its internal sequencer like this:

https://github.com/facebook/jest/blob/master/packages/jest-test-sequencer/src/index.ts#L83-L110

this sequencer has some logic built in to re-run and re-prioritise test runs. This leads to a different order on every single test run. With an alphabetic order we get a more deterministic testrun than with the built in sequencer. It is more reproducible this way. 

The general problem is that we are using jest for blockchain tests, this is not the best technology fit. This is already being mitigated with the `--runInBand` flag sequencing the tests over all.

Where exactly shall i document this? Inline or in a separate document?

Reply:
Thanks for the explanation...
I would document as a comment what you just said...  for future maintainers to know why we are using a diff sequencer.